src/main/webapp/app/exam/exam-scores/exam-scores.component.ts

```diff
@@ -93,6 +95,9 @@ export class ExamScoresComponent implements OnInit, OnDestroy {
     ) {}
 
     ngOnInit() {
+        for (let i = 0; i < 100 / this.binWidth; i++) {
+            this.ngxData.push({ name: i.toString(), value: 0 });
+        }
```

Comment:
maybe add a comment above, this seems kind of magical here... what does it do?

Reply:
I added an explanation in edf3ef8. I hope it makes this statement a bit clearer